grails-app/controllers/org/pih/warehouse/api/PrepaymentInvoiceItemApiController.groovy

```diff
@@ -17,4 +19,11 @@ class PrepaymentInvoiceItemApiController {
         prepaymentInvoiceService.removeInvoiceItem(params.id)
         render status: 204
     }
+
+    def update() {
+        JSONObject jsonObject = request.JSON
+        Integer quantity = jsonObject.get('quantity')
+        prepaymentInvoiceService.updateInvoiceItemQuantity(params.id, quantity)
+        render status: 200
+    }
```

Comment:
btw, from the frontend perspective, update/post methods should return the newly updated state, because we always have to refetch the new data. It's not a change request, because in this one case, on the frontend it's not necessary to fetch the data again, but in most cases, we should.

Reply:
I agree, but here we have a bit more complex update, where we are passing one invoice item, but in reality we are updating two, so should we return both here or only the one that was sent in requests? Due to this, I decided to keep it simple and return only status, at least for now.